oddbird/sass-site
# Get Involved and Community Guideline pages

![](https://source.unsplash.com/featured/?cute,bird&CHANGE_ME)




## Steps to test/reproduce
- Visit /community 
- Visit /community-guidelines
- They should match the current versions of the site
